MediaGalleryUi/view/adminhtml/web/template/grid/columns/image.html

```diff
@@ -0,0 +1,19 @@
+<!--
+    /**
+    * Copyright © Magento, Inc. All rights reserved.
+    * See COPYING.txt for license details.
+    */
+-->
+<div class="media-gallery-image">
+    <div class="masonry-image-block media-gallery-image-block" attr="'data-id': $col.getId($row())">
+        <img attr="src: $col.getUrl($row())" class="media-gallery-image-column" css="$col.getClasses($row())" click="function(){ expandPreview($row()) }" data-role="thumbnail"/>
+    </div>
+</div>
+<div class="masonry-image-description">
+    <ul class="media-gallery-image-details">
+        <li class="name" data-bind="text: $col.getName($row())"></li>
+        <li class="source"><img if="$col.getSource($row())" class="adobe-stock-icon" data-bind="attr: {src: $col.getSource($row())}" /></li>
```

Comment:
See my comment above, also, I think we can use a condition like `ko if: $row().source_icon_url ` to display source icon only if it is specified

Reply:
ok